deploy/operator.yaml

```diff
@@ -182,6 +182,8 @@ spec:
         - /sroperator
         args:
         - --leader-elect
+        - --zap-time-encoding iso8601
+        - --zap-encoder console
```

Comment:
This parameters are broken.. If I try to install the operator using this `deploy/operator.yaml` I am receiving this error:
```bash
kubectl logs kube-starrocks-operator-b49c6d4fc-sljzj -f -p 
flag provided but not defined: -zap-time-encoding iso8601
Usage of /sroperator:
```

To have success to install the operator, I need to remove the both lines:
```bash
        - --zap-time-encoding iso8601
        - --zap-encoder
```

Reply:
Thanks, I will try to fix it.